releasenotes/notes/0.19/qdrift-as-product-formula-044a37a106a45a47.yaml

```
@@ -1,19 +1,16 @@
 ---
 features:
   - |
-    Reformulate the former :class:`~qiskit.opflow.evolutions.trotterizations.QDrift`
-    as a synthesis method instead of a :obj:`~qiskit.opflow.evolutions.TrotterizationBase`. The
-    other available synthesis methods are
-
-    * :class:`~qiskit.synthesis.LieTrotter` - first order Trotterization
-    * :class:`~qiskit.synthesis.SuzukiTrotter` - higher order Trotterization
-    * :class:`~qiskit.synthesis.MatrixExponentiation` - exact, matrix-based evolution
+    The :class:`~qiskit.opflow.evolutions.QDrift` class was reformulated as a
+    synthesis method for :obj:`.PauliEvolutionGate`, deriving from
+    :obj:`~qiskit.opflow.evolutions.TrotterizationBase`.
 
     .. code-block:: python
```

Because this finishes with a print, I assume the intention was to execute it.

```suggestion
    .. jupyter-execute::
```

---

I've been moving away from `jupyter-execute` in the release notes a little bit, because by definition release notes go out-of-date but stay in the documentation, so they'll often stale.  I probably changed the directive and forgot to remove the `print`.  The important bit here is the creation of the `PauliEvolutionGate`.

---

Done in e624bd6.

---

>  [...] by definition release notes go out-of-date but stay in the documentation, so they'll often stale.
 
This is a really good point. I think is reasonable to stop using execute in release note as a policy.